types/nodemark/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "private": true,
+  "dependencies": {}
+}
```

Comment:
this file serves no purpose, mind removing it?

Reply:
adopted the suggestion